src/screens/Booking/Booking.styles.ts

```diff
@@ -0,0 +1,10 @@
+export default {
+  container: {
+    width: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  scrollview: {
+    width: '100%',
+  },
+};
```

Comment:
let's fix the TS warning in the file below src/screens/Booking/Booking.tsx doing it:

```suggestion
import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  container: {
    width: '100%',
    alignItems: 'center',
    justifyContent: 'center',
  },
  scrollview: {
    width: '100%',
  },
});

```